v4l2-request-test.c

```diff
@@ -122,20 +122,17 @@ static void print_summary(struct config *config, struct preset *preset)
 	printf(" Height:       %d\n", preset->height);
 	printf(" Frames count: %d\n", preset->frames_count);
 
-	printf(" Format: ");
+	printf(" Codec Type:   ");
```

Comment:
Replacing Format with Codec is definitely a change I would take in, but adding more than the short names for the codecs seems overkill. If you'd like to improve this area, one thing would be to use proper names: MPEG-2, H.264 and H.265 instead.

Reply:
Will produce an update.